<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="logicGateCanvasFrame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>571</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="logicGateListFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="MenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-3</y>
      <width>800</width>
      <height>661</height>
     </rect>
    </property>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>-180</y>
       <width>271</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
       <pointsize>30</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>logicolypse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newGameButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>-120</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start a New Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chapterButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>-90</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chapters</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>-60</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quit Game :(</string>
     </property>
    </widget>
    <widget class="QWidget" name="levelWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>-320</y>
       <width>321</width>
       <height>281</height>
      </rect>
     </property>
     <widget class="QPushButton" name="levelSixButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>105</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Level Six</string>
      </property>
     </widget>
     <widget class="QPushButton" name="levelFiveButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>105</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Level Five</string>
      </property>
     </widget>
     <widget class="QPushButton" name="levelThreeButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>105</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Level Three</string>
      </property>
     </widget>
     <widget class="QPushButton" name="levelTwoButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>105</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Level Two</string>
      </property>
     </widget>
     <widget class="QPushButton" name="levelOneButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>105</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Level One</string>
      </property>
     </widget>
     <widget class="QPushButton" name="levelFourButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>105</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Level Four</string>
      </property>
     </widget>
     <widget class="QLabel" name="levelLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>25</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Choose a Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>121</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>‚Üê Back to Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Tutorial</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
